### ISSUE TITLE
Guide creation UX issues

### ISSUE THREAD

**Issue by toribr** [cityofaustin/techstack#4359]
Because only LIVE links can be added to a guide, it makes it really hard to see what you are doing.

Therefore:

As an author, i want to be able to link to pages that are still drafts
As a resident, I want no broken links ever

from @briaguya:
If we could figure out a way to allow linking to drafts, and then use publish preflight to make sure everything we link to goes live on publish that’d be huge.
Then we could have a guide creation flow where all the pages are drafted and added to the guide, and on publish we have messaging saying “publishing this guide will also publish these 6 pages, continue?” or something.

**Comment by Niicck** [comment]
An idea: could we have a a different UI for draft pages? Like they're a different background color with a message? We could allow users to add draft pages, with the expectation that they won't publish. Publish Preflight would check that at least 1 page is live.

Here's another tricky UX thing that I've been thinking about. What if your linked page gets unpublished? What happens to the guide? Should we have guardrails that prevent you from unpublishing a page if its connected to other pages? We haven't run into this yet because I don't think we've really unpublished pages/topics in production.

**Comment by toribr** [comment]
Yep - we've been putting this off, and should include content in the discussion. I'm happy to facilitate that conversation, but it'd be helpful to know what our timeline for addressing this is. IE, how soon should that conversation take place? Would next sprint be soon enough?
